inst/tutorials/vaccines/vaccines.Rmd

```diff
@@ -0,0 +1,332 @@
+  doses <- aggregate(cbind(doses[1:4]),
+                      by = list(doses$month), 
+                      FUN = sum)
+  doses <- as.matrix(t(doses[, -1]))
+  
+  barplot(doses, xlab = "Month",
+          ylab = "Number of doses",
+          col = cols[1:6], space = 0, axes = T,
+          beside = FALSE, xaxs = "i", yaxs = "i",
+          ylim = c(0, max(colSums(doses))*1.1))
+  grid(lty = 2, col = "grey80", lwd = 0.5);box()
+  axis(side = 1, lty = 1, col = "black", pos = 0)
+  legend("topleft", box.lty = 0, legend = c("Dose 1","Dose 2","Dose 3","Booster 1"),
+         fill = cols[1:6], bg="transparent", cex = 0.8, y.intersp = 1.5)
+}
+
+# Control parameters and model output
+sim_length <- 3 * year
+human_population <- 10000
+starting_EIR <- 20
+
+simparams <- get_parameters(list(
+    human_population = human_population,
+    clinical_incidence_rendering_min_ages = 0,
+    clinical_incidence_rendering_max_ages = 5 * year,
+    individual_mosquitoes = FALSE
+  )
+)
+
+simparams <- set_equilibrium(parameters = simparams, init_EIR = starting_EIR)
+
+# Run a model with no interventions in a setting with no seasonality
+output_control <- run_simulation(timesteps = sim_length * 2, parameters = simparams)
+```
+
+## More exercises: Vaccination
+
+Now that you have been introduced to malariasimulation, we will review a few vaccine use cases that might be helpful. This tutorial is largely based on the malariasimulation vaccine [vignette](https://mrc-ide.github.io/malariasimulation/articles/Vaccines.html)-- refer to this for more practice! 
+
+We can use malariasimulation to implement a theoretical transmission blocking vaccine (TBV) or a pre-erythrocytic vaccine. Vaccines can be administered using either a mass vaccination strategy or an age-based strategy. 
+
+To carry this out, we can utilize the functions `set_pev_epi()` or `set_mass_pev().` For the purposes of this training, we will focus on the administration of a pre-erythrocytic vaccine (IE, RTS,S or R21).
+
+First, let's look at the vaccine efficacy parameters for RTS,S. These are saved within malariasimulation:
+
+```{r efficacy_profile, exercise = TRUE}
+
+malariasimulation::rtss_profile # Parameters for the first three doses of RTS,S
+
+malariasimulation::rtss_booster_profile # Parameters for the fourth (booster) dose of RTS,S
+```
+
+These parameters are used to model the relationship between antibody titre and vaccine efficacy, based on previous modelling work conducted in the [group](https://pubmed.ncbi.nlm.nih.gov/37080831/). The meaning of these parameters can be found [here](https://mrc-ide.github.io/malariasimulation/reference/create_pev_profile.html) -- you will likely not need to modify them directly, but know that these parameters impact the magnitude and duration of vaccine impact. 
+
+```{r quiz1, echo= FALSE}
+
+quiz(caption = "Vaccine Efficacy Question",
+     question_numeric(
+       "What is the vaccine efficacy of the first three doses of the RTS,S vaccine?",
```

Comment:
```suggestion
       "What is the maximum vaccine efficacy of the first three doses of the RTS,S vaccine?",
```

Reply:
Looks like this has been implemented as recommended.